src/main/webapp/WEB-INF/view/product/all-products.jsp

```diff
@@ -26,12 +26,12 @@
             </td>
             <td>
                 <a href="${pageContext.request.contextPath}/cart/products/add?id=${product.id}">To cart</a>
-                <a href="${pageContext.request.contextPath}/products/delete?id=${product.id}">Delete</a>
             </td>
         </tr>
     </c:forEach>
 </table>
 <a href="${pageContext.request.contextPath}/products/add">Add product</a>
```

Comment:
is user able to add new products to storage?

Reply:
done